app/models/user.rb

```diff
@@ -15,9 +16,35 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  def remember
+    self.remember_token = User.new_token
```

Comment:
self là gì ?

Reply:
self giống như this trong các ngôn ngữ khác, self là biến instance của class hiện tại (mà nó được gọi trong đó )

Reply:
Có bỏ nó được không em ơi? dùng mỗi remember_token không ấy?